google/cloud/bigquery/big_query_read_connection.gcpcxx.pb.h

```diff
@@ -78,11 +69,10 @@ class BigQueryReadConnection {
 };
 
 std::shared_ptr<BigQueryReadConnection> MakeBigQueryReadConnection(
-    BigQueryReadConnectionOptions const& options =
-        BigQueryReadConnectionOptions());
+    internal::Options const& options = internal::Options());
```

Comment:
Just `= {}`. It's simpler, and the type is already mentioned, so there's no need to mention it again.

Also, the argument should be passed by value, because it will be modified by the implementation.